build/tasks/livereload.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
+var livereload = require( "tiny-lr" );
+var server;
+
+module.exports = function( grunt ) {
+	grunt.registerTask( "livereload", function() {
+		if ( !server ) {
+			server = livereload();
+			server.listen( 35729, function() {
```

Comment:
Only thought is to set it up as an argument that you can pass when running `grunt`. Looks like it should be possible without too much difficulty via https://gruntjs.com/api/grunt.option.

I'll look into it soon and update this PR.